finder-app/manual-linux.sh

```diff
@@ -48,33 +68,100 @@ then
 fi
 
 # TODO: Create necessary base directories
+# Creating root file system in output directory and making basic directories
+mkdir "$OUTDIR/rootfs"
+cd "$OUTDIR/rootfs"
+mkdir -p bin dev etc home lib lib64 proc sbin sys tmp usr var
+mkdir -p usr/bin usr/bin usr/sbin
+mkdir -p var/log
+
 
 cd "$OUTDIR"
 if [ ! -d "${OUTDIR}/busybox" ]
 then
-git clone git://busybox.net/busybox.git
-    cd busybox
+git clone https://github.com/mirror/busybox.git
+#git clone git@github.com:mirror/busybox.git
+
+	    cd busybox
     git checkout ${BUSYBOX_VERSION}
     # TODO:  Configure busybox
+    echo "Busy box configuration"
+    make distclean
+    make defconfig
+   
 else
+    echo "Busy box already configured"
     cd busybox
 fi
 
 # TODO: Make and install busybox
+make ARCH=${ARCH} CROSS_COMPILE=${CROSS_COMPILE}
+make CONFIG_PREFIX="${OUTDIR}/rootfs" ARCH=${ARCH} CROSS_COMPILE=${CROSS_COMPILE} install
+
+cd "${OUTDIR}"
 
 echo "Library dependencies"
-${CROSS_COMPILE}readelf -a bin/busybox | grep "program interpreter"
-${CROSS_COMPILE}readelf -a bin/busybox | grep "Shared library"
+${CROSS_COMPILE}readelf -a ./rootfs/bin/busybox | grep "program interpreter"
+${CROSS_COMPILE}readelf -a ./rootfs/bin/busybox | grep "Shared library"
+
+cp /home/bhakti/Downloads/arm-gnu-toolchain-13.3.rel1-x86_64-aarch64-none-linux-gnu/aarch64-none-linux-gnu/libc/lib/ld-linux-aarch64.so.1 ${OUTDIR}/rootfs/lib
```

Comment:
@BhaktiRamani use `-print-sysroot` to find the cross compiler path.  This will be the wrong path for the automated test runner and might be related to your full test failure.